logstash-core/lib/logstash/config/modules_common.rb

```diff
@@ -71,6 +71,10 @@ def self.pipeline_configs(settings)
         .select { |mn| mn != "arcsight" }
         .each { |mn| deprecation_logger.deprecated("The #{mn} module has been deprecated and will be removed in version 9.") }
 
+      if specified_and_available_names.include?("arcsight")
+        deprecation_logger.deprecated("The ArcSight module has been deprecated and will be removed in future version.")
```

Comment:
```suggestion
        deprecation_logger.deprecated("The ArcSight module has been deprecated and will be removed in 9.0.0")
```

Reply:
ArcSight is not scheduled to deprecate in version 9

Reply:
🤦 nevermind my comment, I was just on autopilot